built-in-references/Kubernetes/container-no-privilege-escalation/template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sazurecontainernoprivilegeescalation
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAzureContainerNoPrivilegeEscalation
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
```

Comment:
How is this policy different from the existing policy https://github.com/Azure/azure-policy/blob/master/built-in-references/Kubernetes/container-no-privilege/template.yaml? Do we need another policy?

Reply:
Privilege escalation is different from a privileged container. It is possible to have an unprivileged container that has permissions to escalate those privileges